librtmp/amf.h

```diff
@@ -32,6 +32,12 @@
 #define FALSE	0
 #endif
 
+#ifdef EXPORT
+#define DECL __declspec(dllexport)
```

Comment:
Just curious, does this change make the lib Windows only or can it be compiled for other platforms too?

Reply:
Got it. Thanks!